File: pinyougou/pinyougou-manage-web/src/main/webapp/js/service/brandService.js
app.service("brandService", function ($http) {
    //查询品牌列表并分页
    this.findPage = function (pageNo, rows) {
        return $http.get("../brand/findPage.do?pageNo=" + pageNo + "&rows=" + rows);
    };
    //添加
    this.add = function (entity) {
        return $http.post("../brand/add.do", entity);
    };
    //根据id查询
    this.findOne = function (id) {
        return $http.get("../brand/findOne.do?id=" + id);
    }
    //修改
    this.update = function (entity) {
        return $http.post("../brand/update.do", entity);
    };
    //删除
    this.delete = function (ids) {
        return $http.get("../brand/delete.do?ids=" + ids);
    };
    //按条件查询
    this.search = function (entity, pageNo, rows) {
        return $http.post("../brand/search.do?pageNo=" + pageNo + "&rows=" + rows, entity);
    };
    //查询品牌列表
    this.selectOptionList = function () {
        return $http.post("../brand/selectOptionList.do");
    }
});

File: pinyougou/pinyougou-sellergoods/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import com.pinyougou.service.BaseService;
import com.pinyougou.vo.PageResult;

import java.util.List;
import java.util.Map;

public interface BrandService extends BaseService<TbBrand> {
    public List<TbBrand> queryAll();

    PageResult search(TbBrand brand, Integer pageNo, Integer rows);

    //批量删除
    public void batchDelete(Long[] ids);

    /**
     * 查询品牌列表
     *
     * @return
     */
    List<Map<String, String>> selectOptionList();
}
